docs/source/users/pmodel/pmodel_details/pmodel_overview.md

```diff
@@ -26,43 +25,86 @@ reference documentation](../../../api/pmodel_api)
+
+A P Model can then be fitted to calculate the expected optimal light use efficiency
+given the environment. The basic calculation captures how many moles of carbon can be
+captured for each mole of photosynthetically active photons captured by the leaf. In its
+simplest form, this equation is:
+
+$$
+  \text{LUE} = M_C \cdot \phi_0 \cdot m_j,
+$$
+
+where $M_C$ is the molar mass of carbon and $\phi_0$ (the quantum yield efficiency of
+photosynthesis) captures how many moles of photons are required to capture one mole of
+carbon dioxide.
+
+The term $m_j$ is at the heart of the P model and describes the trade off between
+[carbon dioxide capture and water loss in photosynthesis](./optimal_chi). Given the
+environmental conditions, a leaf will adjust its stomata, setting a ratio of internal to
+external carbon dioxide partial pressure ($\chi$) that optimises this trade off. Under
+adverse conditions, this limits the partial pressure of carbon dioxide within the leaf
+($c_i = c_a \chi$) and gives rise to the limitation term $m_j$ that describes the
+resulting loss in light use efficiency. This calculation also allows the P model to
+estimate the intrinsic water use efficiency of photosynthesis (IWUE) as micromoles per
+mole of photons.
+
+There are several methods for the calculation of $\chi$ and $m_j$, which are selected
+using the `method_optchi` argument when fitting a P model. These methods specify the
+choice of C3 and C4 photosynthetic pathways but also environmental modification of
+optimal $\chi$.
+
+The P model can optionally include further limitations to the light use efficiency of
+photosynthesis. The extended equation shows two modifications:
+
+$$
+  \text{LUE} = M_C \cdot \phi_0(E) \cdot  m_j \cdot f_J
+$$
+
+* The function $\phi_0(E)$ captures methods that introduce environmental modulation of
+  the [quantum yield efficiency of photosynthesis](./quantum_yield), such as variation
+  in $\phi_0$ with air temperature. These methods are selected using the `method_kphio`
+  argument when fitting a P model.
+
+* The additional term $f_j$ describes further [limitation of the electron transfer
+  rate](./jmax_limitation) of photosynthesis ($J_{max}$ limitation). These methods are
+  selected using the `method_jmaxlim` argument when fitting a P model.
+
+:::{warning}
+
+Several of the approaches implemented within the P model in `pyrealm` may be estimating
+the same underlying environmental limitation of light use efficiency via different
+pathways. As an example, there are multiple approaches to incorporating effects of [soil
+moisture stress](soil_moisture) on productivity, via modulation of $\phi_0$, $m_j$ and
+the calculation of GPP penalty factors.
+
+Some combinations of methods may therefore result in multiple corrections for the same
+limitation. At present, `pyrealm` does not automatically detect such over-correction, so
+take care when selecting methods for fitting the P model.
```

Comment:
I think we should add this as an issue. It shouldn't be too difficult to have a compatibility check for any combination of selected methods. We could also add a quick reference to the docs for users.